redaxo/src/core/pages/system.settings.php

```diff
@@ -90,6 +95,20 @@
     $sel_lang->addOption($l, $l);
 }
 
+// see https://github.com/filp/whoops/blob/master/docs/Open%20Files%20In%20An%20Editor.md
+$whoopsConfig = rex::getProperty('whoops');
+$whoopsEditor = isset($whoopsConfig['editor']) ? $whoopsConfig['editor'] : '';
+$supportedEditors = ['' => rex_i18n::msg('system_editor_no_editor'), 'emacs' => 'Emacs', 'idea' => 'IDEA', 'macvim' => 'MacVim', 'phpstorm' => 'PhpStorm (macOS only)', 'sublime' => 'Sublime Text 2 and 3', 'textmate' => 'Textmate', 'xdebug' => 'Xdebug via xdebug.file_link_format (php.ini)', 'vscode' => 'Visual Studio Code', 'atom' => 'Atom'];
```

Comment:
"phpstorm" kann man scheinbar auch in Windows und Linux nutzen, man braucht aber zusätzlich
https://github.com/aik099/PhpStormProtocol
bzw.
https://github.com/sanduhrs/phpstorm-url-handler

(wie ich gerade bei symfony gelesen habe: https://symfony.com/doc/current/reference/configuration/framework.html#ide)